### ISSUE TITLE
[Feature] Ability to specify a file path to save replays to.

### ISSUE THREAD

**Issue by Crinfarr** [Zingabopp/OBSControl#8]
I know I could use OBS's built in directory chooser, but I use OBS for more than one thing and would appreciate this upgrade.

**Comment by Zingabopp** [comment]
So a folder specified in the mod's config to switch OBS to? As in, you always want your Beat Saber videos to go to that specific folder?

**Comment by Crinfarr** [comment]
Yes, exactly like that.  I've been using NVidia's overlay to record and it created ~/Videos/Beat Saber/*, and it would be nice to have my recordings saved there.

**Comment by Zingabopp** [comment]
Been awhile, but this option *should* be coming soon:tm: in the next version.
